Mathlib/Util/Simp.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2025 Paul Lezeau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Lezeau
+-/
+import Lean.Meta.Tactic.Simp.Types
+import Qq
+
+/-! # Additional simp utilities
+
+This file adds additional tools for metaprogramming with the `simp` tactic
+
+-/
+
+open Lean Meta Qq
+
+namespace Simp
+
+open Simp
+
+/-- `Qq` version of `Simp.Methods.discharge?`, which avoids having to use `~q` matching
```

Comment:
Can you give the full name of `discharge?` here so that it ends up linked in the docs?